### PR TITLE
PROD-1978 : The groups in the message screen and notification are using the default avatar when the group avatar is disabled

### PR DESCRIPTION
**Pull request by milanhirapra**
### Jira Issue:

https://buddyboss.atlassian.net/browse/PROD-1978

### General Note
Keep all conversations related to this PR in the associated Jira issue(s). Do NOT add comment on this PR or edit this PR’s description.

### Notes to Developer
* Ensure the IDs (i.e. PROD-1) of all associated Jira issues are reference in this PR’s title
* Ensure that you have achieved the Definition of Done before submitting for review
* When this PR is ready for review, move the associate Jira issue(s) to “Needs Review” (or “Code Review” for Dev Tasks)

### Notes to Reviewer
* Ensure that the Definition of Done have been achieved before approving a PR
* When this PR is approved, move the associated Jira issue(s) to “Needs QA” (or “Approved” for Dev Tasks)

### CODE REVIEW

File: src/bp-messages/bp-messages-functions.php [hunk 1]
```diff
@@ -1181,9 +1181,12 @@ function bp_messages_get_avatars( $thread_id, $user_id ) {
 
 		if ( 'group' === $message_from && 'all' === $message_users && 'open' === $message_type ) {
 			if ( bp_is_active( 'groups' ) ) {
-				$group_name   = bp_get_group_name( groups_get_group( $group_id ) );
-				$group_avatar = array(
-					'url'  => bp_core_fetch_avatar(
+
+				$group_name       = bp_get_group_name( groups_get_group( $group_id ) );
+				$group_avatar_url = bb_get_buddyboss_group_avatar();
```
Comment (chetansatasiya) [inline src/bp-messages/bp-messages-functions.php:1186]:
```suggestion
				$group_avatar_url = '';
```

File: src/bp-messages/bp-messages-functions.php [hunk 2]
```diff
@@ -1193,9 +1196,14 @@ function bp_messages_get_avatars( $thread_id, $user_id ) {
 							'title'      => $group_name,
 							'html'       => false,
 						)
-					),
+					);
+				}
```
Comment (chetansatasiya) [inline src/bp-messages/bp-messages-functions.php:1200]:
```suggestion
				} else {
				     $group_avatar_url = bb_get_buddyboss_group_avatar();
				}
```

File: src/bp-templates/bp-nouveau/includes/messages/ajax.php [hunk 3]
```diff
@@ -895,6 +895,8 @@ function bp_nouveau_ajax_get_user_message_threads() {
 			$group_message_thread_type = bp_messages_get_meta( $last_message_id, 'group_message_thread_type', true );
 			$group_message_fresh       = bp_messages_get_meta( $last_message_id, 'group_message_fresh', true );
 
+			$group_avatar = bb_get_buddyboss_group_avatar();
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/includes/messages/ajax.php:898]:
```suggestion
			$group_avatar = '';
```

File: src/bp-templates/bp-nouveau/includes/messages/ajax.php [hunk 4]
```diff
@@ -903,28 +905,30 @@ function bp_nouveau_ajax_get_user_message_threads() {
...
+							'item_id'    => $group_id,
+							'object'     => 'group',
+							'type'       => 'full',
+							'avatar_dir' => 'group-avatars',
+							'alt'        => sprintf( __( 'Group logo of %s', 'buddyboss' ), $group_name ),
+							'title'      => $group_name,
+							'html'       => false,
+						)
+					);
+				}
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/includes/messages/ajax.php:920]:
```suggestion
				} else {
				    $group_avatar = bb_get_buddyboss_group_avatar();
				}
				
```

File: src/bp-templates/bp-nouveau/includes/messages/ajax.php [hunk 5]
```diff
@@ -948,31 +952,37 @@ function bp_nouveau_ajax_get_user_message_threads() {
 			$group_id      = ( isset( $first_message->id ) ) ? (int) bp_messages_get_meta( $first_message->id, 'group_id', true ) : 0;
 
 			if ( $group_id ) {
+
+				$group_avatar = bb_get_buddyboss_group_avatar();
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/includes/messages/ajax.php:956]:
```suggestion
				$group_avatar = '';
```

File: src/bp-templates/bp-nouveau/includes/messages/ajax.php [hunk 6]
```diff
@@ -948,31 +952,37 @@ function bp_nouveau_ajax_get_user_message_threads() {
...
+								'item_id'    => $group_id,
+								'object'     => 'group',
+								'type'       => 'full',
+								'avatar_dir' => 'group-avatars',
+								'alt'        => sprintf( __( 'Group logo of %s', 'buddyboss' ), $group_name ),
+								'title'      => $group_name,
+								'html'       => false,
+							)
+						);
+					}
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/includes/messages/ajax.php:974]:
```suggestion
					} else {
					       $group_avatar = bb_get_buddyboss_group_avatar();
					}
```

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
Need requested changes

**Review by chetansatasiya — APPROVED** [review]

**Review by KartikSuthar — APPROVED** [review]

**Pull request merged**